### DISCUSSION TITLE
Is it possible to upload files directly to Filament Table by dragging the file to a column/row via Drag And Drop?

### DISCUSSION THREAD

**Discussion by alexworldbr** [filamentphp/filament#13688] [Ideas]
Hi guys,

Is it possible to upload files directly to Filament Table by dragging the file to a column/row via Drag And Drop or input/select icon?

Has anyone ever done this?

Thanks,

Alex
from Brazil

**Comment by alexworldbr** [comment]
Anybody?

**Reply by alexworldbr** [reply]
Up

**Reply by leandrocfe** [reply]
No, I guess. It would be a custom column..
